core/__tests__/models/option.ts

```diff
@@ -311,7 +311,7 @@ describe("models/option", () => {
           });
         });
 
-        afterAll(async () => {
+        beforeAll(async () => {
```

Comment:
I'm curious about this change- with two `beforeAll` hooks that mess with `connection`, what order do these run in? It seems like before this was cleaning up the destination after running the tests but I'm not sure what the intention is now. I'm reading that the destination is being created and destroyed in this `beforeAll` step, but that's clearly not what's happening since tests are passing 🤔

Reply:
Yeah... that's a mistake.  I'll update!